examples/05_transformers.ipynb

```diff
@@ -0,0 +1,851 @@
+    "* reviewing common data container formats for series and panels\n",
+    "* showcasing the signature of transformers that transform a series or panel\n",
+    "* showcasing the signature of transformers that transform a pair ot series to a scalar, e.g., distances or kernels\n",
+    "* how to search `sktime` for transformers of a certain type"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.1 Data container format<a class=\"anchor\" id=\"section_1_1\"></a>\n",
+    "\n",
+    "`sktime` transformers apply to individual time series and panels (= collections of time series).\n",
+    "This is formalized as abstract \"scientific types\" `Series` and `Panel`, with multiple possible in-memory representations, so-called \"mtypes\".\n",
+    "\n",
+    "For the purpose of this tutorial, we will be working with the most common mtypes. For more details and formal data type specifications, see the \"datatypes and datasets\" tutorial.\n",
+    "\n",
+    "`Series` are commonly represented as:\n",
+    "\n",
+    "* `pandas.Series` for univariate time series and sequences\n",
+    "* `pandas.DataFrame` for uni- or multivariate time series and sequences\n",
+    "\n",
+    "The `Series.index` and `DataFrame.index` are used for representing the time series or sequence index. `sktime` supports pandas integer, period and timestamp indices.\n",
+    "\n",
+    "`Panel`-s are commonly represented as:\n",
+    "* a `pandas.DataFrame` in a specific format, defined by the `pd-multiindex` scitype - this has a double row index, for time points and instances\n",
+    "* a `list` of `pandas.DataFrame`, where all `pandas.DataFrame` are in the `Series` format. The different `list` elements are the different instances\n",
+    "\n",
+    "In either case, the \"time\" index must be a `sktime` compatible time index type, as for `Series`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sktime.datatypes import get_examples"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-04-10T16:07:05.849641Z",
+     "iopub.status.busy": "2021-04-10T16:07:05.849188Z",
+     "iopub.status.idle": "2021-04-10T16:07:06.049023Z",
+     "shell.execute_reply": "2021-04-10T16:07:06.049499Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# example of a univariate series\n",
+    "get_examples(\"pd.Series\", \"Series\")[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# example of a multivariate series\n",
+    "get_examples(\"pd.DataFrame\", \"Series\")[1]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# example of a panel in pd-multiindex format\n",
```

Comment:
maybe use the term mtype more throughout the notebook, so a user gets used to the term. I think this is important because we *do* use the word mtype as a parameter in many of these functions, so it is good for a user to be exposed to that word more in the tutorial.

```suggestion
    "# example of a panel with mtype pd-multiindex\n",
```